tests/Generator/ODMGeneratorTest.php

```diff
@@ -17,6 +17,9 @@
 use Sonata\MediaBundle\Generator\ODMGenerator;
 use Sonata\MediaBundle\Tests\Entity\Media;
 
+/**
+ * @deprecated
```

Comment:
a test is public API, it cannot be deprecated. Let's mark it as covering a legacy path.
```suggestion
 * @group legacy
```